exampleSite/config.yaml

```diff
@@ -0,0 +1,60 @@
+baseURL: "https://example.com/"
+languageCode: "en-us"
+title: "Example Site"
+theme: scientific-python-hugo-theme
+relativeURLs: true
+disableKinds: ["taxonomy"]
+
+markup:
+  highlight:
+    noClasses: false
+  goldmark:
+    renderer:
+      unsafe: true
+    renderHooks:
+      link:
+        enableDefault: true
+
+params:
+  author:
+    name: "Example Team"
+  description: "Example"
+  images:
+    - /images/logo.svg
+  search: true
+  navbarlogo:
+    image: logo.svg
+    text: Scientific Python Hugo Theme
+    link: /
+  fonts:
+    - name: "Lato"
+      weights: [400, 900]
+  hero:
+    title: Example Site
+    image: logo.svg
+    buttontext: Get Started
+    buttonlink: "#"
+  navbar:
+    - title: User Guide
```

Comment:
Maybe another link here? Or at least one that resolves to something meaningful.

Reply:
We can clean this up later. For now, I just want to make sure we aren't asking folks to copy a ton of stuff they need to delete.